### DISCUSSION TITLE
Read Deploy Hook Trigger Payload

### DISCUSSION THREAD

**Discussion by gabbibrooks** [vercel/community#906] [Idea]
Hi all,

I currently have a few apps deployed on the platform and set up some deploy hooks to connect my prismic cms to trigger deploys for our nuxt static site. Per the documentation from prismic [here](https://prismic.io/docs/technologies/webhooks#the-webhook-payload), they mention that a payload is sent with every webhook. Is there any way to read or access this payload from a deploy hook when a deployment is triggered? As far as I can tell, there's no way to see what actually triggered a deployment from the deployment logs.

My use case is that when a deployment is triggered from prismic documents being published, I would like to view which documents were published from the webhook payload.

**Comment by RealSebbooo** [comment]
Hey @brooksztb,

have you got any information how you can do this so far?

**Reply by gabbibrooks** [reply]
I haven't no, but I also haven't looked further into it any further as my work priorities have shifted away from this.
